text/0000-compressed-nft-standard.md

```diff
@@ -0,0 +1,195 @@
+- **TEP**: [0](https://github.com/ton-blockchain/TEPs/pull/0)
+- **title**: Compressed NFT Standard
+- **status**: Draft
+- **type**: Contract Interface
+- **authors**: [Maxim Gromov](https://github.com/krigga), [Narek Abovyan](https://github.com/naltox), [Ivan Nedzvetsky](https://github.com/stels-cs), [Daniil Sedov](https://github.com/Gusarich)
+- **created**: 28.07.2023
+- **replaces**: -
+- **replaced by**: -
+
+# Summary
+
+Extension for [NFT Standard](https://github.com/ton-blockchain/TEPs/blob/master/text/0062-nft-standard.md).
+
+A standard interface for non-fungible token collections which allow cheap mass creation of items/tokens.
+
+# Motivation
+
+There is a need (to create large communities, big advertising campaigns, etc) for a type of NFT collection which allows its authors/owners to create large amounts of items at low cost.
+
+# Guide
+
+A straightforward way to achieve properties described in [Motivation](#motivation) is using [Merkle trees](https://en.wikipedia.org/wiki/Merkle_tree) - then each item/token is a leaf of the tree and can be "claimed" by providing a small proof of its presence in the tree, and the collection itself only needs to store a single hash for all items (not per item).
+
+### Useful links
+
+1. [Reference contract implementation](https://github.com/ton-community/compressed-nft-contract)
+2. [Reference augmenting API implementation](https://github.com/ton-community/compressed-nft-api)
+
+# Specification
+
+> The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+## Data types
+
+### Item data
+Item data is an object that completely represents a particular item. MUST include fields `metadata` (an object that MUST include fields `owner` - the address that will own the item once claimed; and `individual_content` - the individual content Cell that must be passed to the collection's `get_nft_content` get method; MAY include other fields) and `index` - the index of this particular item; MAY include other fields.
+
+Example:
+```json
+{
+	"metadata": {
+		"owner": "UQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAJKZ", // user-friendly url-safe address string
+		"individual_content": "te6cckEBAQEAAgAAAEysuc0=" // base64 BoC serialized Cell
+	},
+	"index": 0
```

Comment:
is index should always be integer? what if index is 256-bit long like in domains?